linera-core/src/client/mod.rs

```diff
@@ -2054,7 +2052,9 @@ where
         block.check_proposal_size(max_size, &blobs)?;
         self.state_mut().set_pending_block(block.clone());
 
-        Ok(HashedCertificateValue::new_confirmed(executed_block))
+        Ok(Hashed::new(ConfirmedBlock::from_validated(
+            ValidatedBlock::new(executed_block),
+        )))
```

Comment:
```suggestion
        Ok(Hashed::new(ConfirmedBlock::new(executed_block))
```

(We probably don't need the other `from_validated` function anymore either?)

Reply:
Yeah, I can remove it now.

Reply:
Done in [5774032](https://github.com/linera-io/linera-protocol/pull/2988/commits/57740329ac8f5bf9d001f9899c9c59be78acda09)